README.md

```diff
@@ -22,6 +22,18 @@ It will give you access to this information in object-oriented Ruby, as JSON, or
 
 ## Usage
 
+### Command line interface
+
+`./bin/far [output] [NUMBER]`
```

Comment:
I'd make this an argument (e.g., `--body`, `--extract`) rather than a subcommand (e.g. `far body`)